src/glossary.rst

```diff
@@ -2800,6 +2854,14 @@ defines the compile- and run-time semantics of a program.
 :dp:`fls_xd997kd2i73a`
 See :s:`Item`.
 
+.. _fls_2yPpT2kPT5ys:
+
+item signature
+^^^^^^^^^^^^^^
+
+:dp:`fls_jZ7vlrcEDDLg`
+An :dt:`item signature` is ??? HELP LUKAS!
```

Comment:
This is solely used by `Inferred Type`. I think we might want to rewrite the legaility rule there as item signature is not such a good term after all

Reply:
I have removed this definition and expanded the legality rule of `Inferred Type`, please have a look at a856074